GetFed/GetFed/Helpers/Identity.swift

```diff
@@ -12,6 +12,10 @@ enum Identity: String {
     case homeVC
     
     case homeToFoodSearchSegue
+    
+    case foodSearchResultCell
+    
+    case FoodResultTableViewCell
```

Comment:
It's important to be consistent in your naming. Swift enum cases should start with lower case.

Reply:
Thanks! Corrected.